src/test/ui/exhaustive_integer_patterns.rs

```diff
@@ -8,8 +8,9 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-#![feature(exhaustive_integer_patterns)]
```

Comment:
Can you create an additional test file which has the parts that do not need any feature gates at all so we have better assurance that stable behavior isn't regressed?